pthread/pthread.c

```diff
@@ -845,6 +850,8 @@ int pthread_cond_init(pthread_cond_t *__restrict cond, const pthread_condattr_t
 {
 	int err;
 
+	cond->clock_id = attr->clock_id;
```

Comment:
`attr` can be `NULL`, then `cond->clock_id` should have default value

Reply:
Thx, fixed